docker/run.sh

```diff
@@ -86,10 +86,15 @@ function _run_tests() {
 
 function bootstrap() {
     JS_SETUP=yes setup python
-    ./manage.py sync_couch_views
-    ./manage.py migrate --noinput
-    ./manage.py compilejsi18n
-    ./manage.py bootstrap demo admin@example.com password
+    su cchq -c "./manage.py sync_couch_views && \
+                ./manage.py migrate --noinput && \
+                ./manage.py compilejsi18n && \
+                ./manage.py bootstrap demo admin@example.com password"
```

Comment:
To run `migrate` on a clean install, you should set the environment variable `CCHQ_IS_FRESH_INSTALL=1` to skip those manual migrations.  We shouldn't have any migrations which require manual intervention and don't respect that parameter.

https://github.com/dimagi/commcare-hq/blob/ed152ce6942203269d84ddf3653863991a0710e4/README.md#set-up-your-django-environment